<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RowColDeleteInsertDialog</class>
 <widget class="QDialog" name="RowColDeleteInsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert/Delete Rows</string>
  </property>
  <widget class="QTabWidget" name="insertDeleteTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>551</width>
     <height>91</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="insertRowTab">
    <attribute name="title">
     <string>insert row</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>495</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>insert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numRowsWidget">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>rows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="insertRowLocationWidget">
          <item>
           <property name="text">
            <string>below</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>above</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pivotRowLocation">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="insertRowButton">
        <property name="text">
         <string>insert</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>delete row</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>527</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>delete row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="deleteRowWidget">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>272</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="deleteRowButton">
        <property name="text">
         <string>delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>insert column</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>531</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>insert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numColumnsWidget">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>columns </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="insertColumnLocationWidget">
          <item>
           <property name="text">
            <string>right of</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>left of</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>column</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pivotColumnLocation">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="insertColumnButton">
        <property name="text">
         <string>insert</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <property name="accessibleDescription">
     <string/>
    </property>
    <attribute name="title">
     <string>delete column</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>517</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer_20">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>delete column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="deleteColumnWidget">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_19">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>216</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="deleteColumnButton">
        <property name="text">
         <string>delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>insertDeleteTabWidget</tabstop>
  <tabstop>numRowsWidget</tabstop>
  <tabstop>insertRowLocationWidget</tabstop>
  <tabstop>pivotRowLocation</tabstop>
  <tabstop>insertRowButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>deleteRowButton</tabstop>
  <tabstop>numColumnsWidget</tabstop>
  <tabstop>insertColumnLocationWidget</tabstop>
  <tabstop>pivotColumnLocation</tabstop>
  <tabstop>insertColumnButton</tabstop>
  <tabstop>deleteColumnWidget</tabstop>
  <tabstop>deleteColumnButton</tabstop>
  <tabstop>deleteRowWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
